spec/support/factories.rb

```diff
@@ -6,6 +6,13 @@
     password "password"
   end
 
+  factory :admin, class: User do
```

Comment:
You can save some code by updating your user factory rather than having a factory solely for admin. For example, the factory would look like this:

 factory :user do
    sequence(:email) { |n| "personemailllll#{n}@example.com" }
    display_name "Bob"
    password "password"
    admin false
  end

With this factory, your default user has admin set as false, which you can change in the test as I will note above.